xbmc/addons/AddonDll.cpp

```diff
@@ -400,31 +390,5 @@ ADDON_STATUS CAddonDll::TransferSettings()
   return ADDON_STATUS_OK;
 }
 
-bool CAddonDll::CheckAPIVersion(int type)
-{
-  /* check the API version */
-  const char* kodiVersion = kodi::addon::GetTypeVersion(type);
-  const char* addonVersion = m_pDll->GetAddonTypeVersion(type);
```

Comment:
Thought lo leave it in, in case it will be needed in the future and the old addons allow the check.

Because it is made in the headers and the addon is not directly affected.

Thanks for your other comments, I replace the strings there.